db-scripts/src/main/resources/migration.sql

```diff
@@ -1000,3 +1002,21 @@ UPDATE `info` SET `DB_SCHEMA_VERSION`="2.12.15";
 ##version: 2.12.16
 ALTER TABLE `structural_variant` ADD COLUMN `ANNOTATION_JSON` JSON AFTER `SV_STATUS`;
 UPDATE `info` SET `DB_SCHEMA_VERSION`="2.12.16";
+
+-- changes for issue 9461
+ALTER TABLE `mutation_event` DROP COLUMN `FUNCTIONAL_IMPACT_SCORE`;
+ALTER TABLE `mutation_event` DROP COLUMN `FIS_VALUE`;
+ALTER TABLE `mutation_event` DROP COLUMN `LINK_XVAR`;
+ALTER TABLE `mutation_event` DROP COLUMN `LINK_PDB`;
+ALTER TABLE `mutation_event` DROP COLUMN `LINK_MSA`;
+
+-- changes for issue 9779
+ALTER TABLE `mutation_event` DROP COLUMN `ONCOTATOR_DBSNP_RS`;
+ALTER TABLE `mutation_event` DROP COLUMN `ONCOTATOR_UNIPROT_ENTRY_NAME`;
+ALTER TABLE `mutation_event` CHANGE COLUMN `ONCOTATOR_REFSEQ_MRNA_ID` `MUTATION_REFSEQ_MRNA_ID` varchar(64);
+ALTER TABLE `mutation_event` CHANGE COLUMN `ONCOTATOR_CODON_CHANGE` `MUTATION_CODON_CHANGE` varchar(255);
+ALTER TABLE `mutation_event` CHANGE COLUMN `ONCOTATOR_UNIPROT_ACCESSION` `MUTATION_UNIPROT_ACCESSION` varchar(64);
+ALTER TABLE `mutation_event` CHANGE COLUMN `ONCOTATOR_PROTEIN_POS_START` `MUTATION_PROTEIN_POS_START` int(11);
+ALTER TABLE `mutation_event` CHANGE COLUMN `ONCOTATOR_PROTEIN_POS_END` `MUTATION_PROTEIN_POS_END` int(11);
+UPDATE `info` SET `DB_SCHEMA_VERSION`="2.12.16";
```

Comment:
@jagnathan Based on the database versioning we discussed, we should update the version to `2.13.0`: https://github.com/cBioPortal/cbioportal/blob/master/docs/Database-Versioning.md